Getting-Started-with-Akka-http-signature.md

```diff
@@ -215,10 +211,10 @@ write(wd/"publicKey.ttl", toTurtle)
 
 ### Attaching Public keys to a File/URI
```

Comment:
### Publishing the Public key to the Web

There are a number of ad hoc ways to do this and one simple standard way to publish it.
The ad hoc ways requires placing the public key on the server by either copying it over to the right location with ftp, sftp, scp, ... or of connecting to the remote server using telnet, or ssh and building the object there. In each case this requires knowing a lot in addition to the location (URL) one wants the document to be found at. It requires knowing:
  - the type of server that is running
  - which files are metadata files, their conventions and usually their syntax
  - how to make sure content negotiation is working correctly to avoid tying semantic web URIs to representations,
  - where the root of the file system is
  - what the password or access control system is on that server
  - and usually getting access to the server via ssh gives access to all the conentent there, rather than allowing
    access control rules to be set per resource.

For illustration see the document [ I pointed you to twice already] and to the setup for rww-play.